packages/react-scripts/utils/createJestConfig.js

```diff
@@ -20,8 +20,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     moduleNameMapper: {
-      '^.+\\.(?!(js|jsx|css|json)$)[^\\.]+$': resolve('config/jest/FileStub.js'),
```

Comment:
Thanks for clarification, I think I took the commit comment's mention of `.css` too literally. This does seem to make more sense.